src/record/polygon.rs

```diff
@@ -383,6 +383,15 @@ impl<PointType: HasXY> From<GenericPolyline<PointType>> for GenericPolygon<Point
 /// ( collection of [Point](../point/struct.Point.html))
 pub type Polygon = GenericPolygon<Point>;
 
+impl Polygon {
+    /// Returns an opaque representation of the polygon that implements the
+    /// `geo_traits::MultiPolygon` trait.
+    #[cfg(feature = "geo-traits")]
+    pub fn into_geo_trait(self) -> Result<crate::geo_traits_impl::MultiPolygon, crate::Error> {
```

Comment:
Perhaps `into_geo_traits` would be better wording when the crate is `geo-traits` 🤷‍♂️

Reply:
Ah, thanks for catching!

Reply:
and maybe add a `try_` prefix since it returns a result :thinking: